R.ODAF.utils/R/write_additional_output.R

```diff
@@ -88,5 +88,5 @@ write_additional_output <- function(count_data,
         col.names = TRUE)
     }
   }
-
+  return(biomarkers)
```

Comment:
Might be useful at some point to include something like
`return(list(biomarkers, bmdexpress))`

Just a thought - not necessary right now

Reply:
Good thought! Maybe something to return to when (if?) we start trying to do BMC modelling as a module?